amr-wind/equation_systems/icns/source_terms/GeostrophicForcing.cpp

```diff
@@ -31,6 +31,10 @@ GeostrophicForcing::GeostrophicForcing(const CFDSim&)
         amrex::Real rot_time_period = 86400.0;
         pp.query("rotational_time_period", rot_time_period);
         coriolis_factor = 2.0 * utils::two_pi() / rot_time_period;
+
+        amrex::Real latitude = 90.0;
+        pp.query("latitude", latitude);
+        AMREX_ALWAYS_ASSERT(amrex::Math::abs(latitude - 90.0) < 1.0e-14);
```

Comment:
While we are at it can we also add an information message that prints out the `f` 

```c++
amrex::Print() << "Geostrophic forcing: Coriolis factor = " << coriolis_factor << std::endl;
```